taskcluster/ci/fetch/kind.yml

```diff
@@ -63,3 +63,12 @@ tasks:
             url: https://github.com/getsentry/sentry-cli/releases/download/2.10.0/sentry-cli-Windows-x86_64.exe
             sha256: 3d9e04ab5f1bea8709f8302a33e81b75aca0a93d1431dde56063549a335acfd0
             size: 10361344
+    macos-miniconda:
+        description: MiniConda3 osx-x86 Python 3.10 
+        fetch:
+            type: static-url 
+            url: https://repo.anaconda.com/miniconda/Miniconda3-py310_22.11.1-1-MacOSX-x86_64.sh
+            sha256: 7406579393427eaf9bc0e094dcd3c66d1e1b93ee9db4e7686d0a72ea5d7c0ce5
+            artifact-name: miniconda.sh
+            size: 46574107
+        fetch-alias: miniconda-osx
```

Comment:
nit: Newline. I suggest turning on the "Insert Final Newline" setting on VSCode so it does this automatically.

Reply:
Great suggestion! - I'll add this to the .vscode pr!

Reply:
While you are at it, maybe also add the "Trim Trailing Whitespace" option :pleading_face:

Reply:
Done :)